pinot-segment-local/src/main/java/org/apache/pinot/segment/local/utils/TableConfigUtils.java

```diff
@@ -313,6 +321,65 @@ private static void validateTaskConfigs(TableConfig tableConfig) {
     }
   }
 
+  private static void validateMergeRollupTaskConfig(TableTaskConfig tableTaskConfig, Schema schema) {
+    if (tableTaskConfig != null && tableTaskConfig.isTaskTypeEnabled(MinionConstants.MergeRollupTask.TASK_TYPE)) {
+      Map<String, String> taskTypeConfig = tableTaskConfig.getConfigsForTaskType(MinionConstants.MergeRollupTask.TASK_TYPE);
+      Map<String, Map<String, String>> levelToConfigMap = MergeRollupTaskUtils.getLevelToConfigMap(taskTypeConfig);
+      boolean validateAggregateFunction = false;
+      for (Map.Entry<String, Map<String, String>> levelToConfigEntry : levelToConfigMap.entrySet()) {
+        String mergeLevel = levelToConfigEntry.getKey();
+        Map<String, String> mergeConfig = levelToConfigEntry.getValue();
+        String mergeType = mergeConfig.get(MinionConstants.MergeRollupTask.MERGE_TYPE_KEY);
+        Preconditions.checkState(
+            mergeType != null && (mergeType.equalsIgnoreCase("CONCAT") || mergeType.equalsIgnoreCase("ROLLUP")),
```

Comment:
Instead of hard-coding `CONCAT, ROLLUP`, I think that the following it better:

```
MergeType type;
try {
    type = MergeType.valueOf(mergeType.toUpperCase())
} catch (IllegalArgumentException e) {
    // fail
}
```
Otherwise, we need to add here when we add the new merge type.